runtime/compiler/p/codegen/PPCJNILinkage.cpp

```diff
@@ -115,10 +115,16 @@ TR::Register *J9::Power::JNILinkage::buildDirectDispatch(TR::Node *callNode)
    uintptr_t targetAddress;
 
    bool crc32m1 = (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_update);
-   bool crc32m2 = ((callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes)  ||
-                  (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes0));
-   bool crc32m3 = ((callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateByteBuffer) ||
-                  (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateByteBuffer0));
+   bool crc32m2 = (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes)
+#if JAVA_SPEC_VERSION >= 9
+                  || (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes0)
+#endif
+                  ;
+   bool crc32m3 = (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateByteBuffer)
+#if JAVA_SPEC_VERSION >= 9
+                  || (callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateByteBuffer0)
+#endif
+                  ;
```

Comment:
Can you rewrite this to look something like

```
bool crc32m2 =
#if JAVA_SPEC_VERSION >=9
    callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes0
#else
    callSymbol->getRecognizedMethod() == TR::java_util_zip_CRC32_updateBytes
#endif
    ;
bool crc32m3 = ...
```

so that it is consistent with the other checks you added?